### PR TITLE
Fix error in github pages publication pipeline

### PR DESCRIPTION
**Pull request by bess**
Also, remove an unused stray file

### CODE REVIEW

File: .tool_versions
Comment (carolyncole) [inline .tool_versions:1]: @bess is this required?  I utilize this file with asdf, so I will not be in sync with the versions.  What file now shows the tool versions?

**Review by carolyncole — CHANGES_REQUESTED** [review]
@bess if the changes to remove tool versions are required then we need to make sure everyone is utilizing the tool versioning software you are utilizing that doe snot use that file.

File: .tool_versions
Comment (bess) [reply .tool_versions:1]: @carolyncole Sorry, I should have explained why I removed this file. This project, for some reason, has a `.tool_versions` file (note the underscore), in addition to the `.tool-versions` file (note the dash). This PR removes only the file with the underscore, which I think must have been added by mistake at some point.

Comment (carolyncole) [reply .tool_versions:1]: Thanks!  I missed the _ vs -

**Review by carolyncole — APPROVED** [review]

**Pull request merged**